src/client_networking.rs

```diff
@@ -55,49 +55,23 @@ impl From<StrictError> for ConnectionError {
 }
 
 
-// I'd change the declaration to: request_table(table_name: &str, server_address: &str)
-// Agree with name => table_name but this gets a csv. Should be called download_csv, though, to be consistent with server()
```

Comment:
The first line was me, the second was you. 

Okay, that can make sense to me, but what I was more trying to get at was that the underlying data structure may not be relevant to the caller of this function. I haven't dived into the `StrictTable` type yet. But it seems to me that the `csv` data format it something that's used for storage and transportation and is a "layer of abstraction" below the `StrictTable` itself that's being returned by this function. To me it'd make more sense if you had other functions requesting tables by different formats maybe